drivers/performance_measurement/app/views/performance_measurement/warehouse_reports/reports/_form.haml

```diff
@@ -1,5 +1,5 @@
 - url = polymorphic_path(@report.report_path_array)
-- alert = 'Please note, the following options are additive.  If you choose a single project and a project type, the report will run for all projects in the project type and the chosen project.  If you choose a data source and project type, the report will run for all projects in the chosen data source and chosen project type. Projects are limited by supported Project Types (PH, ES, TH, SH, SO) and selected CoC.'
+- alert = 'Included projects are limited by supported types (ES, SO, SH, TH, PH) and selected CoC.'
```

Comment:
```suggestion
- alert = 'Included projects are limited by those in the SPM (ES, SO, SH, TH, PH), selected CoCs, and selected Funding Sources.'
```

Reply:
The performance measurement report only supports selecting one CoC. Changed to "*the* selected CoC" for clarity